src/app/ApplicationTemplate.Presentation/Framework/Review/ReviewCondition.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+/// <summary>
+///  A condition used to determine if a review should be requested based on <see cref="ReviewSettings"/>.
+/// </summary>
+/// <param name="ct"><see cref="CancellationToken"/>.</param>
+/// <param name="reviewPromptSettings">The persisted <see cref="ReviewSettings"/> tracked by the application.</param>
+/// <param name="currentDateTime">The current date and time.</param>
+/// <returns>If a review should be requested.</returns>
+public delegate Task<bool> ReviewCondition(CancellationToken ct, ReviewSettings reviewPromptSettings, DateTimeOffset currentDateTime);
```

Comment:
This is named like a class and not like a delegate. I suggest you either switch it to an interface like IReviewCondition or rename the deletage.

Reply:
Should we add the suffix `Delegate`?